config/samples/resources/cloudbuildworkerpool/workerpool-with-peered-network/serviceusage_v1beta1_service.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: serviceusage.cnrm.cloud.google.com/v1beta1
+kind: Service
```

Comment:
I think in general we don't want to add Service unless it's a different service other than the ones resources we've already created in this sample need.

In this case, because we've created ServiceNetworkingConnection, we should assume users already enabled the Service Networking API.
If we add Service Networking API, for consistency purpose, we'll add Service resources for Cloud Build, Compute APIs as well, which is kinda redundant.

If somehow to make this sample work, users need to enable BigQuery API, then it's reasonable to add a BigQuery service here.

Reply:
It is niche to include the service (a workable, out of the box sample). But I'm fine to remove it.